src/github.com/kegsay/gitpher/github_api.go

```diff
@@ -5,21 +5,20 @@ import (
 	"net/http"
 )
 
-// GithubUserContentURL The base URL to Github User Content
-const GithubUserContentURL = "https://raw.githubusercontent.com/"
+const githubUserContentURL = "https://raw.githubusercontent.com/"
 
-// GetGithubUserContent Fetch a raw file from Github.
-func GetGithubUserContent(contentPath string) (string, error) {
-	githubURL := GithubUserContentURL + contentPath
+// GetGithubUserContent fetches a raw file from Github.
+func GetGithubUserContent(contentPath string) ([]byte, error) {
+	githubURL := githubUserContentURL + contentPath
 	response, err := http.Get(githubURL)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
 	defer response.Body.Close()
 	body, err := ioutil.ReadAll(response.Body)
```

Comment:
It's a trade-off, right? I don't know the implementation details of HttpResponse.Body, but my guess is that either you leave the TCP socket open and unread (by returning response.Body), or you keep the whole response in memory (by reading to a []byte).